src/Spice86/Views/DebugWindow.axaml

```diff
@@ -1,519 +1,310 @@
-                                <DataGridTextColumn Header="Memory Segment" Binding="{Binding MemorySegment}" />
-                                <DataGridCheckBoxColumn Header="Stack Instruction?" Binding="{Binding IsStackInstruction}" />
-                                <DataGridCheckBoxColumn Header="Is IP relative?" Binding="{Binding IsIPRelativeMemoryOperand}" />
-                                <DataGridTextColumn Header="IP relative memory address" Binding="{Binding IPRelativeMemoryAddress}" />
-                                <DataGridTextColumn Header="Bytes" Binding="{Binding Bytes}" />
-                                <DataGridTextColumn Header="Length" Binding="{Binding Length}" />
-                                <DataGridTextColumn Header="FlowControl" Binding="{Binding FlowControl}" />
+                                <DataGridTextColumn Binding="{Binding Address}" Header="Address" />
+                                <DataGridTextColumn Binding="{Binding SegmentedAddress}" Header="Based on CS:IP" />
+                                <DataGridCheckBoxColumn Binding="{Binding IsCsIp}" Header="Is at CS:IP?" />
+                                <DataGridTextColumn Binding="{Binding Converter={StaticResource InstructionToStringConverter}}" Header="Disassembly (MASM)" />
+                                <DataGridTextColumn Binding="{Binding IP16}" Header="16-bit IP" />
+                                <DataGridTextColumn Binding="{Binding IP32}" Header="32-bit IP" />
+                                <DataGridTextColumn Binding="{Binding SegmentPrefix}" Header="Segment Prefix" />
+                                <DataGridTextColumn Binding="{Binding MemorySegment}" Header="Memory Segment" />
+                                <DataGridCheckBoxColumn Binding="{Binding IsStackInstruction}" Header="Stack Instruction?" />
+                                <DataGridCheckBoxColumn Binding="{Binding IsIPRelativeMemoryOperand}" Header="Is IP relative?" />
+                                <DataGridTextColumn Binding="{Binding IPRelativeMemoryAddress}" Header="IP relative memory address" />
+                                <DataGridTextColumn Binding="{Binding Bytes}" Header="Bytes" />
+                                <DataGridTextColumn Binding="{Binding Length}" Header="Length" />
+                                <DataGridTextColumn Binding="{Binding FlowControl}" Header="FlowControl" />
                             </DataGrid.Columns>
                             <TextElement.FontFamily>
-                                <OnPlatform Default="{StaticResource RobotoMonoFont}">
-                                    <On Options="Windows" Content="{StaticResource ConsolasFont}" />
-                                </OnPlatform>
+                                <OnPlatform Default="{StaticResource RobotoMonoFont}" />
                             </TextElement.FontFamily>
```

Comment:
Isn;t the TextElement.FontFamily already set to this right before the DataGrid?

Reply:
Fixed.